sdk/src/rpc/types/request.ts

```diff
@@ -8,6 +8,8 @@ import {
     RPCAppSessionAllocation,
     RPCAllowance,
     GetLedgerTransactionsFilters,
+    RPCAppStateIntent,
+    ProtocolVersion,
```

Comment:
Why some types have `RPC` prefix and some don't?

Reply:
I will rename `ProtocolVersion` to `RPCProtocolVersion`.

Reply:
91fc916ec2c652bc84aa7a066eba91d110c1e00f